lib/plugins/aws/lib/monitorStack.test.js

```diff
@@ -311,6 +508,7 @@ describe('monitorStack', () => {
         StackEvents: [
           {
             EventId: '1a2b3c4d',
+            StackName: 'new-service-dev',
             LogicalResourceId: 'somebucket',
```

Comment:
I think in this case the event would come from the bucket , hence LogicalResourceId should be as it is (i.e. it would contain the name of the logicalID of the bucket, not the stack)